install/recorders.csv

```diff
@@ -1018,8 +1018,7 @@ Kinemetrics,FBA-ES-T-BASALT,BASALT,620,INGS,20,280,,0,0,2015-06-30T05:15:00Z,201
 Kinemetrics,FBA-ES-T-BASALT,BASALT,620,GKBS,20,303,,0,0,2020-09-03T21:13:00Z,9999-01-01T00:00:00Z
 Kinemetrics,FBA-ES-T-BASALT,BASALT,621,NNZ,20,0,,0,0,2012-01-05T23:10:00Z,2013-01-17T21:00:00Z
 Kinemetrics,FBA-ES-T-BASALT,BASALT,621,INZ,20,0,,0,0,2013-06-12T00:10:00Z,2015-05-28T06:00:00Z
-Kinemetrics,FBA-ES-T-BASALT,BASALT,621,TLZ,20,0,,0,0,2016-02-12T01:30:01Z,2020-01-15T02:12:00Z
-Kinemetrics,FBA-ES-T-BASALT,BASALT,621,TLZ,21,0,,0,0,2020-01-15T02:16:00Z,2020-07-02T02:31:00Z
+Kinemetrics,FBA-ES-T-BASALT,BASALT,621,TLZ,21,0,,0,0,2016-02-12T01:30:01Z,2020-01-15T02:12:00Z
```

Comment:
Should the end date of Basalt be `2020-07-02T02:31` at LC 21?
Noting that data exists from January to July, up to when the next recorder (Titan) was installed `2020-07-05`.

Sample data randomly checked: 
```
aws s3 ls --profile sod_readrole s3://geonet-archive/miniseed/2020/2020.032/TLZ.NZ/
2020-02-06 20:44:00    7744512 2020.032.TLZ.21-BNE.NZ.D
2020-02-06 20:43:57    7787520 2020.032.TLZ.21-BNN.NZ.D
2020-02-06 20:43:56    7045120 2020.032.TLZ.21-BNZ.NZ.D

```
```
aws s3 ls --profile sod_readrole s3://geonet-archive/miniseed/2020/2020.099/TLZ.NZ/
2020-04-14 05:54:09    8551424 2020.099.TLZ.21-BNE.NZ.D
2020-04-14 08:52:29    8388608 2020.099.TLZ.21-BNN.NZ.D
2020-04-14 05:54:09    7639552 2020.099.TLZ.21-BNZ.NZ.D
2020-04-14 08:52:29   28896256 2020.099.TLZ.21-HNE.NZ.D
2020-04-14 08:52:29   27369984 2020.099.TLZ.21-HNN.NZ.D
2020-04-13 19:49:38   27885056 2020.099.TLZ.21-HNZ.NZ.D
```
```
aws s3 ls --profile sod_readrole s3://geonet-archive/miniseed/2020/2020.184/TLZ.NZ/
2020-07-06 23:33:45    4366848 2020.184.TLZ.21-HNE.NZ.D
2020-07-06 23:33:46    4312064 2020.184.TLZ.21-HNN.NZ.D
2020-07-07 02:30:45    4393984 2020.184.TLZ.21-HNZ.NZ.D
```

Reply:
of course!